services/terrain/src/terrain/services/permanent_id_requests.clj

```diff
@@ -98,9 +98,11 @@
 (defn- create-publish-dir
   "Creates the Permanent ID Requests publish directory, if it doesn't already exist."
   []
-  (let [publish-path (config/permanent-id-publish-dir)]
+  (let [publish-path (config/permanent-id-publish-dir)
+        curators     (config/permanent-id-curators-group)]
     (when-not (data-info/path-exists? (config/irods-user) publish-path)
-      (data-info-client/create-dirs (config/irods-user) [publish-path]))))
+      (data-info-client/create-dirs (config/irods-user) [publish-path])
+      (data-info/share (config/irods-user) [curators] [publish-path] "write"))))
```

Comment:
I believe this is fine here, but note that this will set the permission to exactly that, which I'm mentioning because it might be mildly unintuitive -- if the curators have ownership this'd revoke that. I think that'd only come up if for some reason the irods user and the curators are the same, so I'll assume it's fine unless you can think of something else.